NetProbe/Model/NetworkMeasurement.mm

```diff
@@ -53,21 +54,69 @@ @implementation NetworkMeasurement
 
 -(id) init {
     self = [super init];
-    self.logLines = [[NSMutableArray alloc] init];
-    self.finished = FALSE;
+    NSBundle *bundle = [NSBundle mainBundle];
+    geoip_asn = [bundle pathForResource:@"GeoIPASNum" ofType:@"dat"];
+    geoip_country = [bundle pathForResource:@"GeoIP" ofType:@"dat"];
+    ca_cert = [bundle pathForResource:@"cacert" ofType:@"pem"];
+    self.completed = FALSE;
     return self;
 }
 
 -(void) run {
     // Nothing to do here
 }
 
--(NSString*) getDate{
+-(NSString*) getDate {
     NSDateFormatter *dateformatter=[[NSDateFormatter alloc]init];
     [dateformatter setDateFormat:@"dd-MM-yyyy HH:mm:ss"];
     return [dateformatter stringFromDate:[NSDate date]];
 }
 
+-(NSString*) getFileName:(NSString*)ext {
+    NSArray *paths = NSSearchPathForDirectoriesInDomains(NSDocumentDirectory, NSUserDomainMask, YES);
+    NSString *documentsDirectory = [paths objectAtIndex:0];
+    //double timestamp = mk::time_now();
+    NSString *fileName = [NSString stringWithFormat:@"%@/test-%@.%@", documentsDirectory, self.test_id, ext];
+    //NSLog(@"fileName %@", fileName);
+    return fileName;
+}
+
+-(void)writeOrAppend:(NSString*)string{
+    NSString *fileName = [self getFileName:@"txt"];
+    NSFileManager *fileManager = [NSFileManager defaultManager];
+    if(![fileManager fileExistsAtPath:fileName])
+    {
+        [string writeToFile:fileName atomically:NO encoding:NSStringEncodingConversionAllowLossy error:nil];
+    }
+    else
+    {
+        NSFileHandle *myHandle = [NSFileHandle fileHandleForWritingAtPath:fileName];
+        [myHandle seekToEndOfFile];
+        [myHandle writeData:[[NSString stringWithFormat:@"\n%@",string] dataUsingEncoding:NSUTF8StringEncoding]];
+    }
```

Comment:
Is it correct to say that here we open the log file and seek to its end every time we have a new log line? Wouldn't it be more natural to open the file once and then write each log line into it?

Would it make sense to add support for writing the log file directly inside MK? (This change would be trivial)